skorch/callbacks/logging.py

```diff
@@ -205,6 +207,76 @@ def on_train_end(self, net, **kwargs):
         if self.close_after_train:
             self.experiment.stop()
 
+class WandbLogger(Callback):
+    """Logs best model and metrics to `Weights & Biases <https://docs.wandb.com/>`_
+
+    "Use this callback to automatically log best trained model and all metrics from
+    your net's history to Weights & Biases after each epoch.
+
+    Examples
+    --------
+    >>> import wandb
+    >>> from skorch.callbacks import WandbLogger
+    >>> wandb_run = wandb.init()
+    >>> wandb.config.update({"learning rate": 1e-3, "batch size": 32})  # optional
+    >>> net = NeuralNet(..., callbacks=[WandbLogger(wandb_run)])
+    >>> net.fit(X, y)
+
+    Parameters
+    ----------
+    wandb_run : wandb.wandb_run.Run
+      wandb Run used to log data.
+
+    save_model : bool (default=True)
+      Saves best trained model.
+
+    keys_ignored : str or list of str (default=None)
+      Key or list of keys that should not be logged to
+      tensorboard. Note that in addition to the keys provided by the
+      user, keys such as those starting with 'event_' or ending on
+      '_best' are ignored by default.
+    """
+
+    # Record if watch has been called previously (even in another instance)
+    _watch_called = False
```

Comment:
Is this really used anywhere? If it is, please move this inside `initialize` and call it `watch_called_`.